onadata/apps/api/viewsets/charts_viewset.py

```diff
@@ -53,21 +62,23 @@ def get_content(self, renderer, data,
         return content
 
 
-class ChartsViewSet(AnonymousUserPublicFormsMixin,
-                    AuthenticateHeaderMixin,
-                    CacheControlMixin,
-                    ETagsMixin,
+# pylint: disable=R0901
+class ChartsViewSet(AnonymousUserPublicFormsMixin, AuthenticateHeaderMixin,
```

Comment:
Pylint `R0901` has to do with "Too many ancestors". 

The formatting at this time was under the mercy of `yapf` command, I tend to take its recommendations. I should probably look into and check-in yapf's configuration.